pkg/controller/controller.go

```diff
@@ -921,7 +911,7 @@ func Run(ctx context.Context, config *Configuration) {
 		}
 	}
 
-	if config.EnableLiveMigrationOptimize && controller.hasKubevirtVMIMigration {
+	if config.EnableLiveMigrationOptimize {
```

Comment:
```suggestion
	if controller.config.EnableLiveMigrationOptimize {
```

统一用 controller.config 吧